docs/naps/2-conda-based-packaging.md

```diff
@@ -0,0 +1,527 @@
+(nap2)=
+
+# NAP-2 — Distributing napari with conda-based packaging
+
+```{eval-rst}
+:Author: Jaime Rodríguez-Guerra, Gonzalo Peña-Castellanos
+:Created: 2022-05-05
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+:Resolved: <date resolved, in yyyy-mm-dd format>
+:Status: Draft
+:Type: <Standards Track | Process>
+:Version effective: <version-number> (for accepted NAPs)
+```
+
+## Abstract
+
+napari can be installed through different means, including `pip` and `conda`. However, users
+not exposed to package managers and command-line interfaces might prefer other options more
+native to their operating system of choice. This is usually achieved through graphical
+installers with a step-by-step interface.
+
+This NAP discusses how we will use and adapt tools borrowed from the `conda` packaging
+ecosystem to build platform-specific installers and implement update strategies for napari and
+its plugin ecosystem.
+
+## Motivation and Scope
+
+napari is packaged for PyPI [^pypi-napari] and conda-forge [^napari-feedstock] and can be
+installed with any client supporting these repositories (e.g., `pip` and `conda`,
+respectively). For a number of releases, platform-specific installers were also provided with
+Briefcase [^briefcase].
+
+Briefcase relies on PyPI packaging to assemble the installer. The PyPI packaging strategy,
+however, presents a series of limitations for the napari ecosystem:
+
+* No standardized building infrastructure. PyPI accepts submissions from any user without
+  requiring any validation or review. As a result, packages can be built using arbitrary
+  toolchains or expecting different libraries in the system. `cibuildwheel` [^cibuildwheel]
+  definitely helps users who want to do it in the right way, but again, there's no guarantee is
+  being used. This can result in ABI incompatibilities with the system (e.g. see issues
+  [^appimage-crash][^appimage-crash2]) and within the plugin ecosystem.
```

Comment:
I cannot agree with this section. This reported error ( #3487) comes from briefcase usage, not pypi. `libffi.so.6` was referred by python itself, not a package. 

Conda constructor just ships with its own version of `libffi.so`. And the briefcase should do the same operation.

Reply:
> With `briefcase`, it's up to their devs to package their own Python, which feels a bit foreign to me.

For the Linux they just bundle python from ubuntu 18.04, but bundle it in the wrong way. Pyinstaller working on same python installation collect all required libraries. 

The idea of PyPi packages is that they will work with any valid python installation.  And I do not meet the problem that comes from valid python installation. But of course, I meet problems with the lack of some package-specific libraries that are shipped from conda.